### PR TITLE
Feature/dx 1000 description non payment token external non payment token

### PR DESCRIPTION
**Pull request by marinafrydenb**
Would like some feedback on the descriptions, as I am not sure I understood it correctly. 
My formatting tool is also not working quite right, so it would be nice with a little help on the tables

### CODE REVIEW

File: _includes/card-operation-paid.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+### Card operation `paid-payment` 
+
+
+{:.code-header}
```
Comment (payex-dev) [inline _includes/card-operation-paid.md:4]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576&bid=18674772)

File: _includes/card-operation-paid.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+| └➔&nbsp;`amount`                  | `integer` | {% include field-description-amount.md %}                                                                                                                                                                    |
+| └➔&nbsp;`tokens`                  | `integer` | List of tokens generated.                                                                                                                                                                                    |
+| └➔&nbsp;`details`                 | `integer` | A human readable and descriptive text of the payment.                                                                                                                                                       | 
+| └─➔&nbsp;`cardBrand`              | `string`  | `Visa`, `MC`, etc. The brand of the card.                                                                                                                                                                    |
```
Comment (payex-dev) [inline _includes/card-operation-paid.md:84]: Codacy Issue found: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

File: _includes/card-operation-paid.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "payment": "/psp/{{ api_resource }}/payments/5adc265f-f87f-4313-577e-08d3dca1a26c",
```
Comment (NullableInt) [inline _includes/card-operation-paid.md:22]:
```suggestion
  "payment": "/psp/{{ api_resource }}/payments/{{ site.payment_id }}",
```

File: _includes/card-operation-paid.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  "paid": {
+    "id": "/psp/{{ api_resource }}/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/paid",
+    "number": 1234567890,
+    "transaction": {
+      "id": "/psp{{ api_resource }}/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/transactions/5adc265f-f87f-4313-577e-08d3dca1a37e",
```
Comment (NullableInt) [inline _includes/card-operation-paid.md:27]:
```suggestion
      "id": "/psp{{ api_resource }}/payments/{{ site.payment_id }}/transactions/{{ site.transaction_id }}",
```

File: _includes/card-authorization-transaction.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+| └─➔&nbsp;`payeeReference`         | `string`  | A unique reference for the transaction.                                                                                                                                                                      |
+| └─➔&nbsp;`failedReason`           | `string`  | The human readable explanation of why the payment failed.                                                                                                                                                    |
+| └─➔&nbsp;`isOperational`          | `bool`    | `true` if the transaction is operational; otherwise `false`.                                                                                                                                                 |
+| └─➔&nbsp;`operations`             | `array`   | The array of operations that are possible to perform on the transaction in its current state.                                                                                                                |
+
+
+## `transactionId` Authorization
```
Comment (asbjornu) [inline _includes/card-authorization-transaction.md:115]: I don't understand this header.

File: _includes/card-authorization-transaction.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/creditcard/payments/{{ page.payment_id }}/authorizations HTTP/1.1
```
Comment (asbjornu) [inline _includes/card-authorization-transaction.md:121]: How is this example different from the previous?

File: _includes/card-operation-paid.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
+### Card operation `paid-payment` 
+
+
```
Comment (asbjornu) [inline _includes/card-operation-paid.md:3]:
```suggestion
```

File: _includes/card-operation-paid.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+{:.code-header}
+**Response**
+
+```http
+GET /psp/creditcard/payments/<paymentId>/paid HTTP/1.1
```
Comment (asbjornu) [inline _includes/card-operation-paid.md:8]:
```suggestion
GET /psp/creditcard/payments/{{ site.payment_id }}/paid HTTP/1.1
```

File: _includes/card-operation-paid.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
+### Card operation `paid-payment`
```
Comment (asbjornu) [inline _includes/card-operation-paid.md:1]: I don't think we need to limit this to Card. We can just call it "Paid Payment" and describe what information the operation gives.

File: payments/card/direct.md [hunk 10]
```diff
@@ -188,6 +188,7 @@ Content-Type: application/json
 | └➔&nbsp;`panToken`                | `string`  | The token representing the specific PAN of the card.                                                                                                                                                         |
 | └➔&nbsp;`panEnrolled`             | `string`  |                                                                                                                                                                                                              |
 | └➔&nbsp;`acquirerTransactionTime` | `string`  | `3DSECURE` or `SSL`. Indicates the transaction type of the acquirer.                                                                                                                                         |
+| └➔&nbsp;`nonPaymentToken`         | `string`  | Result of our own tokenization of the card used. Activated in POS on merchant or merchant group.                                                                                                                                         |
```
Comment (asbjornu) [inline payments/card/direct.md:191]:
```suggestion
| └➔&nbsp;`nonPaymentToken`         | `string`  | Swedbank Pay's tokenization of the card used; identifies the card, but can not be used for payment transactions. Needs to be activated by Swedbank Pay before use.                                                                                                                                         |
```

File: payments/card/other-features.md [hunk 11]
```diff
@@ -29,6 +29,8 @@ covered by the other subsections." %}
 {% include payment-resource.md api_resource="creditcard" documentation_section="card"
 show_status_operations=true %}
 
+{% include card-operation-paid.md %}
```
Comment (asbjornu) [inline payments/card/other-features.md:32]: Why is `paid` documented as a part of this PR? Shouldn't that be done in DX-1017?

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576&bid=18674772)

File: _includes/card-operations.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+| └─➔&nbsp;`id`                     | `string`  | {% include field-description-id.md resource=transaction %}                                                                                                                                                   | 
+| └─➔&nbsp;`number`                 | `string`  | The transaction `number`, useful when there's need to reference the transaction in human communication. Not usable for programmatic identification of the transaction, for that `id` should be used instead.                                                                                                                                                               |
+| └➔&nbsp;`payeeReference`          | `string`  | A unique reference for the transaction.                                                                                                                                                                      | 
+| └➔&nbsp;`orderReference`          | `string(50)`  | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                  |
```
Comment (payex-dev) [inline _includes/card-operations.md:84]: Codacy Issue found: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576&bid=18674772)

File: _includes/card-operations.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+| └➔&nbsp;`amount`                  | `integer` | {% include field-description-amount.md %}                                                                                                                                                                    |
+| └➔&nbsp;`tokens`                  | `integer` | List of tokens generated.                                                                                                                                                                                    |
+| └➔&nbsp;`details`                 | `integer` | A human readable and descriptive text of the payment.                                                                                                                                                       | 
+| └─➔&nbsp;`cardBrand`              | `string`  | `Visa`, `MC`, etc. The brand of the card.                                                                                                                                                                    |
```
Comment (payex-dev) [inline _includes/card-operations.md:88]: Codacy Issue found: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

File: _includes/card-authorization-transaction.md [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+| └─➔&nbsp;`payeeReference`         | `string`  | A unique reference for the transaction.                                                                                                                                                                      |
+| └─➔&nbsp;`failedReason`           | `string`  | The human readable explanation of why the payment failed.                                                                                                                                                    |
+| └─➔&nbsp;`isOperational`          | `bool`    | `true` if the transaction is operational; otherwise `false`.                                                                                                                                                 |
+| └─➔&nbsp;`operations`             | `array`   | The array of operations that are possible to perform on the transaction in its current state.                                                                                                                |
```
Comment (payex-dev) [inline _includes/card-authorization-transaction.md:129]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

File: _includes/card-authorization-transaction.md [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+| {% icon check %} | `transaction`                  | `object`  | The transaction object.                                                         |
+| {% icon check %} | └➔&nbsp;`cardNumber`           | `string`  | Primary Account Number (PAN) of the card, printed on the face of the card.      |
+| {% icon check %} | └➔&nbsp;`cardExpiryMonth`      | `integer` | Expiry month of the card, printed on the face of the card.                      |
+| {% icon check %} | └➔&nbsp;`cardExpiryYear`       | `integer` | Expiry year of the card, printed on the face of the card.                       |
+|                  | └➔&nbsp;`cardVerificationCode` | `string`  | Card verification code (CVC/CVV/CVC2), usually printed on the back of the card. |
+|                  | └➔&nbsp;`cardholderName`       | `string`  | Name of the card holder, usually printed on the face of the card.               |
+
+
+
+:.table .table-striped}
```
Comment (payex-dev) [inline _includes/card-authorization-transaction.md:97]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 2 lines before node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

File: _includes/card-operations.md [hunk 16]
```diff
@@ -0,0 +1,98 @@
+### `paid-payment` Operation
+
+The `paid-payment` operation confirms that the transaction has been successful
+and that the payment is completed. Under `details` you can see which card was
+used to complete the payment. 
+
+
+{:.code-header}
```
Comment (payex-dev) [inline _includes/card-operations.md:8]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576&bid=18674772)

File: _includes/card-operations.md [hunk 17]
```diff
@@ -0,0 +1,98 @@
...
+| └─➔&nbsp;`id`                     | `string`  | {% include field-description-id.md resource=transaction %}                                                                                                                                                   | 
+| └─➔&nbsp;`number`                 | `string`  | The transaction `number`, useful when there's need to reference the transaction in human communication. Not usable for programmatic identification of the transaction, for that `id` should be used instead.                                                                                                                                                               |
+| └➔&nbsp;`payeeReference`          | `string`  | A unique reference for the transaction.                                                                                                                                                                      | 
+| └➔&nbsp;`orderReference`          | `string(50)`  | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                  |
```
Comment (payex-dev) [inline _includes/card-operations.md:84]: Codacy Issue found: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

File: _includes/card-authorization-transaction.md [hunk 18]
```diff
@@ -0,0 +1,202 @@
...
+| {% icon check %} | `transaction`                  | `object`  | The transaction object.                                                         |
+| {% icon check %} | └➔&nbsp;`cardNumber`           | `string`  | Primary Account Number (PAN) of the card, printed on the face of the card.      |
+| {% icon check %} | └➔&nbsp;`cardExpiryMonth`      | `integer` | Expiry month of the card, printed on the face of the card.                      |
+| {% icon check %} | └➔&nbsp;`cardExpiryYear`       | `integer` | Expiry year of the card, printed on the face of the card.                       |
+|                  | └➔&nbsp;`cardVerificationCode` | `string`  | Card verification code (CVC/CVV/CVC2), usually printed on the back of the card. |
+|                  | └➔&nbsp;`cardholderName`       | `string`  | Name of the card holder, usually printed on the face of the card.               |
+
+
+
+{:.table .table-striped}
```
Comment (payex-dev) [inline _includes/card-authorization-transaction.md:169]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

File: _includes/card-authorization-transaction.md [hunk 19]
```diff
@@ -0,0 +1,202 @@
...
+| └─➔&nbsp;`payeeReference`         | `string`  | A unique reference for the transaction.                                                                                                                                                                      |
+| └─➔&nbsp;`failedReason`           | `string`  | The human readable explanation of why the payment failed.                                                                                                                                                    |
+| └─➔&nbsp;`isOperational`          | `bool`    | `true` if the transaction is operational; otherwise `false`.                                                                                                                                                 |
+| └─➔&nbsp;`operations`             | `array`   | The array of operations that are possible to perform on the transaction in its current state.                                                                                                                |
```
Comment (payex-dev) [inline _includes/card-authorization-transaction.md:201]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

File: _includes/card-operations.md [hunk 20]
```diff
@@ -0,0 +1,98 @@
+### Operation `paid-payment`
+
+The `paid-payment` operation confirms that the transaction has been successful
+and that the payment is completed. Under `details` you can see which card was
+used to complete the payment. 
+
+
+{:.code-header}
```
Comment (payex-dev) [inline _includes/card-operations.md:8]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

File: _includes/card-operations.md [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+| └➔&nbsp;`amount`                  | `integer` | {% include field-description-amount.md %}                                                                                                                                                                    |
+| └➔&nbsp;`tokens`                  | `integer` | List of tokens generated.                                                                                                                                                                                    |
+| └➔&nbsp;`details`                 | `integer` | A human readable and descriptive text of the payment.                                                                                                                                                       | 
+| └─➔&nbsp;`cardBrand`              | `string`  | `Visa`, `MC`, etc. The brand of the card.                                                                                                                                                                    |
```
Comment (payex-dev) [inline _includes/card-operations.md:88]: Codacy Issue found: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)

File: _includes/card-authorization-transaction.md [hunk 22]
```diff
@@ -0,0 +1,202 @@
+## Card authorization transaction
+
+The `authorization` resource contains information about an authorization
+transaction made towards a payment. To create a new `authorization` transaction,
+perform a `POST` towards the URI obtained from the `payment.authorization.id`
+from the `payment` resource below. All operations
+that target the payment resource directly produce a response similar to the
+example seen below.
```
Comment (asbjornu) [inline _includes/card-authorization-transaction.md:8]:
```suggestion
from the `payment` resource below. The example is abbreviated for brevity.
```

File: _includes/card-authorization-transaction.md [hunk 23]
```diff
@@ -0,0 +1,202 @@
...
+            "contentType": "application/javascript"
+        },
+        {
+            "method": "GET",
+            "href": "{{ page.front_end_url }}/{{ api_resource }}/payments/authorize/{{ page.transaction_id }}",
+            "rel": "redirect-authorization",
+            "contentType": "text/html"
+        }
+    ]
+}
```
Comment (asbjornu) [inline _includes/card-authorization-transaction.md:77]:
```suggestion
{
    "payment": {
        "id": "/psp/{{ api_resource }}/payments/{{ page.payment_id }}",
        "authorizations": {
            "id": "/psp/{{ api_resource }}/payments/{{ page.payment_id }}/authorizations"
        },
    }
}
```

File: _includes/card-operations.md [hunk 24]
```diff
@@ -0,0 +1,98 @@
+### Operation `paid-payment`
+
+The `paid-payment` operation confirms that the transaction has been successful
+and that the payment is completed. Under `details` you can see which card was
+used to complete the payment. 
+
+
```
Comment (asbjornu) [inline _includes/card-operations.md:7]:
```suggestion
```

File: _includes/card-authorization-transaction.md [hunk 25]
```diff
@@ -0,0 +1,202 @@
...
+| :--------------: | :----------------------------- | :-------- | :------------------------------------------------------------------------------ |
+| {% icon check %} | `transaction`                  | `object`  | The transaction object.                                                         |
+| {% icon check %} | └➔&nbsp;`cardNumber`           | `string`  | Primary Account Number (PAN) of the card, printed on the face of the card.      |
+| {% icon check %} | └➔&nbsp;`cardExpiryMonth`      | `integer` | Expiry month of the card, printed on the face of the card.                      |
+| {% icon check %} | └➔&nbsp;`cardExpiryYear`       | `integer` | Expiry year of the card, printed on the face of the card.                       |
+|                  | └➔&nbsp;`cardVerificationCode` | `string`  | Card verification code (CVC/CVV/CVC2), usually printed on the back of the card. |
+|                  | └➔&nbsp;`cardholderName`       | `string`  | Name of the card holder, usually printed on the face of the card.               |
+
+
+
```
Comment (asbjornu) [inline _includes/card-authorization-transaction.md:168]:
```suggestion
```

File: _includes/card-operations.md [hunk 26]
```diff
@@ -0,0 +1,98 @@
...
+{:.code-header}
+**Response**
+
+```http
+GET /psp/creditcard/payments/<paymentId>/paid HTTP/1.1
```
Comment (asbjornu) [inline _includes/card-operations.md:12]:
```suggestion
GET /psp/creditcard/payments/{{ site.payment_id }}/paid HTTP/1.1
```

**Pull request merged**